gwu-project-1/script.js

```diff
@@ -0,0 +1,15 @@
+let inputText = document.querySelector("#input-text");
+let addBtn = document.querySelector("#add-btn");
```

Comment:
For this, you should use `document.getElementById` because that performs better.

Reply:
I also made changes to this.